docs/UserGuide.adoc

```diff
@@ -505,21 +513,21 @@ __Constraints:__
 // tag::list[]
 ==== Listing tasks : `list` +
 
-To go back to the default list of tasks after a round of filtering, use:
+Want to go back to the default list of task after a search? Simply use the list command:
```

Comment:
*tasks.
Do note that the current version of the UG only uses a lead in line followed by the format. There's no statement that ends with a colon. If you think it's necessary and would be too unclear otherwise, let me know, Otherwise, would appreciate if you could scrap this line.